### PR TITLE
chore(docs): remove `bx` references

### LINKED ISSUE #15132: [Bug]: @carbon/type README updates (bx references)
### Package

@carbon/type

### Browser

Chrome

### Package version

11.25.1

### React version

N/A

### Description

The @carbon/type README needs to be updated from `bx` to `cds`:

```
% grep bx README.md 
| `.bx--type-mono`         | Specify the font face as IBM Plex Mono    |
| `.bx--type-sans`         | Specify the font face as IBM Plex Sans    |
| `.bx--type-serif`        | Specify the font face as IBM Plex Serif   |
| `.bx--type-light`        | Specify the font weight as light (300)    |
| `.bx--type-regular`      | Specify the font weight as regular (400)  |
| `.bx--type-semibold`     | Specify the font weight as semibold (600) |
| `.bx--type-italic`       | Specify the font style as italic          |
| `.bx--type-<type-style>` | Set styles for the given type style       |
```

It was correct for Carbon 10 but it changed in Carbon 11.

### Reproduction/example

N/A

### Steps to reproduce

N/A

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Application/PAL

WKC

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

### PR DESCRIPTION
**Pull request by tw15egan**
Closes https://github.com/carbon-design-system/carbon/issues/15132

#### Changelog

**Changed**

- changes usage of `bx` in docs to `cds`

#### Testing / Reviewing

Ensure no styles are effected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,185 +85,6 @@ check out our [Contributing Guide](/.github/CONTRIBUTING.md) and our
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
 <!-- prettier-ignore-start -->
 <!-- markdownlint-disable -->
-<table>
```
Comment (alisonjoseph) [inline README.md]: Was all of this removed on purpose?

File: packages/react/src/components/FileUploader/FileUploader-story.scss [hunk 2]
```diff
@@ -1,3 +1,3 @@
-.bx--file__selected-file {
+.cds--file__selected-file {
   inline-size: 288px;
 }
```
Comment (alisonjoseph) [inline packages/react/src/components/FileUploader/FileUploader-story.scss]:
```suggestion
```
I think we can remove this.

File: packages/react/src/components/Tile/tile-story.scss [hunk 3]
```diff
@@ -4,6 +4,6 @@
   @include tile.tile($enable-experimental-tile-contrast: true);
 }
 
-div .bx--tile--selectable:not(:last-child) {
+div .cds--tile--selectable:not(:last-child) {
   margin-block-end: 1rem;
 }
```
Comment (alisonjoseph) [inline packages/react/src/components/Tile/tile-story.scss:9]: Just confirming we want this visual change to the Tile stories?

Screenshot 2023-11-17 at 12 27 01 PM

Comment (tw15egan) [reply packages/react/src/components/Tile/tile-story.scss:9]: Just matching the styles we had in the v10 storybook, must have been missed in the migration: https://v7-react.carbondesignsystem.com/?path=/story/components-tile--multi-select

File: packages/react/src/components/FileUploader/FileUploader-story.scss (hunk 2 above)
Comment (tw15egan) [reply packages/react/src/components/FileUploader/FileUploader-story.scss]: Fixed 👍

**Review by tay1orjones — APPROVED** [review]
Thanks for doing this 🙏

**Review by alisonjoseph — APPROVED** [review]
🥳

**Pull request merged**